### PR TITLE
Prepare release v1.34.0

### PR DESCRIPTION
**Pull request by albertteoh**
Prepares Jaeger UI release v1.34.0.

Includes some quality of life improvements for my fellow maintainers:
- Hopefully, simpler process to generate changelogs (ignores dependabot, etc.).
- DRY up release scripts by downloading those defined in jaeger repo, which should make our releases a bit more consistent.

Supersedes https://github.com/jaegertracing/jaeger-ui/pull/1844.

## Checklist
- [x] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [x] I have signed all commits

### CODE REVIEW

**Review by yurishkuro — APPROVED** [review]

File: .prettierignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Ignore generated files
-packages/jaeger-ui/index.d.ts
\ No newline at end of file
+packages/jaeger-ui/index.d.ts
+scripts
```
Comment (yurishkuro) [inline .prettierignore:3]: I think the issue was not with scripts, but with the CHANGELOG that prettier was trying to change (switching between `*` and `-` for bullets)

Comment (yurishkuro) [reply .prettierignore:3]: It's fine to ignore the downloaded scripts, but then I'd narrow it to
```suggestion
scripts/release-notes.py
scripts/draft-release.py
```

Comment (albertteoh) [reply .prettierignore]: Thanks 😄

File: scripts/get-tracking-version.js [hunk 2]
```diff
@@ -1,139 +0,0 @@
-#!/usr/bin/env node
```
Comment (yurishkuro) [inline scripts/get-tracking-version.js:1]: @albertteoh did you delete this by mistake?  I don't am not sure what it is supposed to do, but it is being used during the build https://github.com/jaegertracing/jaeger-ui/blob/f391b862ce143ef32d60947053130c6bdee468c5/packages/jaeger-ui/package.json#L103

Comment (yurishkuro) [reply scripts/get-tracking-version.js:1]: https://github.com/jaegertracing/jaeger-ui/pull/1998

**Pull request merged**